src/views/open-api-docs-view/utils/massage-schema-utils/schema-mod-protobuf-any.ts

```diff
@@ -0,0 +1,82 @@
+import type { OpenAPIV3 } from 'openapi-types'
+import { schemaModComponent } from './schema-mod-component'
+
+/**
+ * Modifies an expected `protobufAny` OpenAPIV3 schema, replacing some
+ * descriptive text in the schema and its properties.
+ *
+ * This is done as we don't want to display the full descriptions of
+ * `protobufAny`, since they were too long in the context of the UI.
+ */
+function shortenProtobufAnyDescription(
```

Comment:
This is the main focus of this PR, the rest is mostly just refactoring to make using this function possible, and to build some consistency in schema modifications across the exist consumers of the OpenAPI docs view.